auth/auth0_manager.go

```diff
@@ -1,23 +1,91 @@
 package auth
 
 import (
-	"log"
+	"fmt"
+	"time"
 
+	"github.com/zpatrick/go-cache"
 	"github.com/zpatrick/rclient"
 )
 
+var timeMultiplier = 1 * time.Second
```

Comment:
This should be an `int`, not a `time.Duration`. A `time.Second` is `1000000000` (see https://golang.org/pkg/time/#pkg-constants). 
This should be `var timeMultiplier = 1`